screens/organizations/index.js

```diff
@@ -57,9 +57,7 @@ export function Organizations() {
               ["Volunteer", "https://www.animal-ethics.org/volunteer/"],
```

Comment:
Shouldn't this links be translated?

If it weren't for the custom links I'd add these orgs info to a list too :smirk:

Reply:
I wanted to keep these links the same, to make it easier for content to be translated. In this case, however, there are language-specific pages available for some of these websites. So I think it makes sense 👍 will pick this up later this week.

Reply:
This has now been fixed!